src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.Runtime.InteropServices
+{
+    public static partial class RuntimeInformation
+    {
+        private const string RuntimeName = "CORECLR";
+        private static Architecture? s_processArch = null;
+        private static bool s_is64BitProcess = IntPtr.Size == 8;
+
+        public static string GetRuntimeDebugName()
+        {
+            return RuntimeName;
```

Comment:
This is a OOB assembly, so it will not ship with Full Framework. 

And on the format of the Runtime string, what is the ```{version}``` you are referring to here, the file version or the marketing version or the nuget version. And what value does knowing the version add, as we have already tried to expose the .NET Framework version in desktop, and it ended up being a constant hardcoded value.

I understand knowing if it's CORECLR or CLR or MRT100 lets you know what framework stack for display info, but I am not convinced why you need version.